lib/elixir_internal_certificate/account/schemas/user.ex

```diff
@@ -16,6 +16,10 @@ defmodule ElixirInternalCertificate.Account.Schemas.User do
     field :password, :string, virtual: true, redact: true
     field :hashed_password, :string, redact: true
 
+    has_many :user_searches,
```

Comment:
some small things

1/ Let alias the `ElixirInternalCertificate.Scrapper.Schemas.UserSearch` 

2/ We no need to specify the `foreign_key: user_id` there

```
alias ElixirInternalCertificate.Scrapper.Schemas.UserSearch

has_many :user_searches, :UserSearch
```